guides/source/error_reporting.md

```diff
@@ -158,3 +175,19 @@ module MySdk
   end
 end
 ```
+
+When using the Rails error reporter, errors may be reported multiple times. For instance, calling `Rails.error.record` will report the error via your subscriber, then raise it. If you have other error mechanisms like a Rack middleware, they will then capture the exception and report it again, then raise it once more, where it may go through your subscriber a second time.
+
+You should adjust your report functionality so it skips reporting an exception it has seen before. One way to do this is to set a custom instance variable on the exception object:
+
+```ruby
+module MySdk
+  class Reporter
+    def report_error(e, **options)
+      return if e.instance_variable_get(:@___myservice_reported)
+    
+      e.instance_variable_set(:@___myservice_reported, true)
```

Comment:
I think I'll integrate a similar mechanism, so that we don't have to suggest that kind of stuff.